lang/cs/Org.Apache.REEF.Network/Group/Driver/Impl/GroupCommunicationMessage.cs

```diff
@@ -46,7 +36,7 @@ private GroupCommunicationMessage()
             string source,
             string destination,
             T message)
-            : base(groupName, operatorName, source, destination)
+            : base(groupName, operatorName, source, destination, typeof(T))
```

Comment:
I mean if base class is a generic of T, you might not need this type parameter. Never mind, I am fine with this change.